pkg/appfile/dryrun/dryrun.go

```diff
@@ -206,3 +210,33 @@ func (d *Option) PrintDryRun(buff *bytes.Buffer, appName string, comps []*types.
 	}
 	return nil
 }
+
+// ExecuteDryRunWithOverridePolicy is similar to ExecuteDryRun function, but patch components with override policy
+func (d *Option) ExecuteDryRunWithOverridePolicy(ctx context.Context, application *v1beta1.Application) (map[string][]*types.ComponentManifest, error) {
+
+	compsForEechOverridePolicy := make(map[string][]*types.ComponentManifest)
```

Comment:
Is "Eech" one typo? I guess you want "Each".

Reply:
I am re-implementing this feature, btw, thanks for the head-ups